test/CMakeLists.txt

```diff
@@ -1539,14 +1539,12 @@ if(iodaconv_bufr_query_ENABLED)
                             gdas.t12z.aircft_AMDAR103.tm00.nc ${IODA_CONV_COMP_TOL}
                     DEPENDS bufr2ioda.x )
 
-#  ecbuild_add_test( TARGET  test_iodaconv_prepbufr_ncep_aircftprofiles2ioda
-#                    TYPE    SCRIPT
-#                    COMMAND bash
-#                    ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
-#                            netcdf
-#                            "${CMAKE_BINARY_DIR}/bin/bufr2ioda.x testinput/bufr_ncep_prepbufr_aircft.yaml"
-#                            gdas.t12z.acft_profiles.prepbufr.nc ${IODA_CONV_COMP_TOL}
-#                    DEPENDS bufr2ioda.x )
+   ecbuild_add_test( TARGET  test_iodaconv_prepbufr_ncep_aircftprofiles2ioda
+                     TYPE    SCRIPT
+                     COMMAND bash
+                     ARGS    ldd
+                             ${CMAKE_BINARY_DIR}/bin/bufr2ioda.x testinput/bufr_ncep_prepbufr_aircft.yaml
```

Comment:
I am not following why this part of the test.

Reply:
I'm abusing your build servers to get extra information about what is going on...

Reply:
Just from test output (ncdump), it seems that instead of setting the presure values that were missing to either 0 or in most cases NaNf. 
It's happening under clang ang gcc so it's probably not a spack issue unless spack is not building bufrlib correctly..
The main problem is that spack gives different hashes to builds that are identical.

Reply:
just looking at the mouns of output from ncdump _all_ the pressure values are missing. However, in you output _most_ are either 0 or nan.. Hope that gives you some bread crumbs.